workers/js-worker-txt.js

```
@@ -180,8 +185,22 @@ function bail (msg) {
  * @param {string} fileName
  * @returns {LambdaFunction}
  */
-function dynamicLambdaRequire (fileName) {
-  return /** @type {LambdaFunction} */ (require(fileName))
+async function dynamicLambdaRequire (fileName) {
+  try {
+    return /** @type {LambdaFunction} */ (require(fileName))
+  } catch (err) {
+    if (err.code === 'ERR_REQUIRE_ESM') {
+      return await import(fileName)
+    }
+
+    if (err.name === 'SyntaxError') {
+      try {
+        return await import(fileName)
+      } catch (_) { }
```

need to rethrow this _ error too

---

oops yeah good catch